interfaces/automated-testing/scd/scd.yaml

```diff
@@ -35,6 +35,416 @@ components:
+
+    ImplicitSubscriptionParameters:
+      description: >-
+        Information necessary to create a subscription to serve a single
+        operational intent's notification needs.
+      type: object
+      required:
+      - uss_base_url
+      properties:
+        uss_base_url:
+          description: >-
+            The base URL of a USS implementation of the parts of the USS-USS API
+            necessary for receiving the notifications that the operational
+            intent must be aware of.  This includes, at least, notifications for
+            relevant changes in operational intents.
+          anyOf:
+          - $ref: '#/components/schemas/SubscriptionUssBaseURL'
+        notify_for_constraints:
+          description: >-
+            True if this operational intent's subscription should trigger
+            notifications when constraints change. Otherwise, changes in
+            constraints should not trigger notifications.  The scope
+            utm.constraint_processing is required to set this flag true, and a
+            USS performing the constraint processing role should set this flag
+            true.
+          type: boolean
+          default: false
+    Priority:
+      description: >-
+        Ordinal priority of the operational intent, as defined by the
+        regulator.  Operational intents with lesser values are lower priority
+        than all operational intents with greater values.  A lower-priority
+        operational intent may not create a conflict with a higher-priority
+        operational intent.  A higher-priority operational intent may create a
+        conflict with a lower-priority operational intent.  The regulator
+        specifies whether an operational intent may create a conflict with other
+        operational intents of the same priority.
+      type: integer
+      default: 0
+    ################################################################################
+    ####################### End of ASTM-standard definitions #######################
+    ################################################################################
+
+    OperationalIntentTestInjection:
+      description: >-
+        Parameters that define an operational intent: this injection is used to
+        create a operational intent reference in the DSS and also responding to 
+        requests for details of that operational intent (by other USSes or the 
+        test driver). The USS under test will need to process this data to both
+        create a valid operational intent reference and responding to a query for  
+        details. 
+      required:
+      - extents
+      - state
+      - priority
+      - volumes
+      - off_nominal_volumes
+      
+      type: object
+      properties:
+        state:
+          $ref: '#/components/schemas/OperationalIntentState'
+        priority:
+          $ref: '#/components/schemas/Priority'
+        volumes:
+          description: >-
```

Comment:
How about just: "Nominal volumes, as would be reported by a USS's operational_intents endpoint"  (and just "Off-nominal volumes, as would be reported by a USS's operational_intents endpoint" for `off_nominal_volumes`) -- we don't want to be redundant where we don't have to, and pointing to that endpoint also gives a hint for how to use this data.

Reply:
Thank you, I have now resolved this.